doc/rst/meta/modules/standard.rst

```diff
@@ -19,7 +19,6 @@ default:
    Errors <standard/Errors>
    Types <standard/Types>
    A subset of the IO symbols <standard/ChapelIO>
-   A subset of the Math symbols <standard/AutoMath>
```

Comment:
I'd probably have this link directly to the auto-included math symbols that have been included in the Math module instead of removing it completely.  That way someone looking at this header won't think there aren't any math symbols included by default.

Reply:
I have been trying to find a way to add this back, but I think we are limited by sphinx. Best I can tell, `toctree` seems to only take filenames and cannot take an arbitrary cross reference

Reply:
Hrm, that's a shame.  I'd probably just list the Math module a second time and let people read for what's automatically included, then